cdap-docs/build.sh

```diff
@@ -171,18 +174,23 @@ function build_docs_outer_level() {
 ################################################## current
 
 function build_all() {
+  echo "========================================================================="
   echo "Building GitHub Docs."
-  ./build.sh docs-github $ARG_2 $ARG_3
+  echo "========================================================================="
+  run_command docs-github-part $ARG_2 $ARG_3
```

Comment:
I'm wondering about this: when I wrote this, I thought that I should duplicate the invocation to prevent any carry-over of environment. Is the environment going to be set the same, run this way? We don't have anything testing that Github google analytics are set correctly, for instance. (That could be added as a test, btw.)